patch/kernel/archive/rockchip-rk3588-6.11/0026-RK3588-Add-VPU121-H.264-Decoder-Support.patch

```diff
@@ -270,23 +310,22 @@ Signed-off-by: Jianfeng Liu <liujianfeng1994@gmail.com>
 Tested-by: Hugh Cole-Baker <sigmaris@gmail.com>
 Signed-off-by: Sebastian Reichel <sebastian.reichel@collabora.com>
 ---
- arch/arm64/boot/dts/rockchip/rk3588s.dtsi | 22 ++++++++++
- 1 file changed, 22 insertions(+)
+ arch/arm64/boot/dts/rockchip/rk3588-base.dtsi | 21 ++++++++++
+ 1 file changed, 21 insertions(+)
 
-diff --git a/arch/arm64/boot/dts/rockchip/rk3588s.dtsi b/arch/arm64/boot/dts/rockchip/rk3588s.dtsi
+diff --git a/arch/arm64/boot/dts/rockchip/rk3588-base.dtsi b/arch/arm64/boot/dts/rockchip/rk3588-base.dtsi
 index 111111111111..222222222222 100644
---- a/arch/arm64/boot/dts/rockchip/rk3588s.dtsi
-+++ b/arch/arm64/boot/dts/rockchip/rk3588s.dtsi
-@@ -1282,6 +1282,28 @@ power-domain@RK3588_PD_SDMMC {
+--- a/arch/arm64/boot/dts/rockchip/rk3588-base.dtsi
++++ b/arch/arm64/boot/dts/rockchip/rk3588-base.dtsi
+@@ -1122,6 +1122,27 @@ power-domain@RK3588_PD_SDMMC {
  		};
  	};
  
 +	vpu121: video-codec@fdb50000 {
-+		compatible = "rockchip,rk3588-vpu121", "rockchip,rk3399-vpu";
++		compatible = "rockchip,rk3588-vpu121", "rockchip,rk3568-vpu";
```

Comment:
`rockchip,rk3399-vpu` has already disabled h264 decoder for you.

Reply:
Yes but it enables JPEG encoder. We already have separate node for the encoder

Reply:
So this core is actually Hantro G1+H1, right?

Reply:
Yes, same as rk3399.